### PR TITLE
Expand list command to take optional multiple Name parameters

### LINKED ISSUE #1398: Add command to show overview of description
It's hard to get an overview of a description, e.g. what scopes it contains, if the scopes are extracted, when it was last modified, from which host it was inspected, etc.

`machinery list` shows at least a good part of this information, but only for all descriptions at once. Adding `machinery list NAME` to show only the overview for the given description would solve the problem.

### PR DESCRIPTION
**Pull request by keberle94**
Fixes #1398

### CODE REVIEW

**Comment by keberle94** [comment]
Closing until Integration test is fixed.

File: man/machinery-list.1.md [hunk 1]
```diff
@@ -4,13 +4,15 @@
 ### SYNOPSIS
 
 `machinery list`
+    NAME
```
Comment (tgoettlicher) [inline man/machinery-list.1.md]: This doesn't say that you can pass multiple arguments.

File: NEWS [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Machinery Release Notes
 
+* Expand `list` command be able to take system descriptions which should be shown
```
Comment (thardeck) [inline NEWS]: I would prefer something like
```
Allow limiting the `list` command output to certain system descriptions by passing them along as argument
```

File: man/machinery-list.1.md [hunk 3]
```diff
@@ -26,6 +28,10 @@ scopes for each system.
 
 ### EXAMPLES
 
+  * Lists the two specified system descriptions `a` and `b`
```
Comment (tgoettlicher) [inline man/machinery-list.1.md]: Colon (`:`) is missing.

File: lib/cli.rb [hunk 4]
```diff
@@ -608,11 +608,15 @@ def self.check_port_validity(port)
 
   desc "List system descriptions"
   long_desc <<-LONGDESC
-    List system descriptions and their stored scopes.
+    List system descriptions and their stored scopes, when no NAME parameter is specified.
```
Comment (tgoettlicher) [inline lib/cli.rb]: `List all system descriptions...`

File: man/machinery-list.1.md (hunk 1 above)
Comment (keberle94) [inline man/machinery-list.1.md]: I just looked that up in the description for our remove command. What would be the right approach here [NAME , NAME] ?

**Comment by keberle94** [comment]
Squashed by #1418

**Pull request closed without merging**